packages/sympy/test_sympy.py

```diff
@@ -9,3 +9,15 @@ def test_sympy(selenium):
     c = sympy.sqrt(a**2 + b**2)
 
     assert c.subs({a: 3, b: 4}) == 5
+
+
+@run_in_pyodide(packages=["sympy", "python-flint"])
+def test_sympy_and_python_flint(selenium):
+    import sympy
+    from sympy.external.gmpy import GROUND_TYPES
+
+    assert GROUND_TYPES == "flint"
```

Comment:
As python-flint is not a dependency of sympy, there is a chance that python-flint is not built when running this test (which will result in a failure). I would suggest updating the test code as follows:

```python
from conftest import package_is_built

def skip_if_not_installed(packages):
    for package in packages:
        if not package_is_built(package):
            pytest.skip(f"{package} not built")

def test_sympy_and_python_flint(selenium):
    skip_if_not_installed(["sympy", "python-flint"])
    
    @run_in_pyodide(packages=["sympy", "python-flint"])
    def run(selenium):
        import sympy
        from sympy.external.gmpy import GROUND_TYPES
    
        assert GROUND_TYPES == "flint"
    
        # Use python-flint for factorisation:
        x = sympy.symbols("x")
        assert (x**2 - 1).factor() == (x + 1) * (x - 1)

    run()
```

Reply:
Good point. I've added that.